packages/docs/src/playground/Button.svelte

```diff
@@ -30,7 +31,7 @@
     block={values.block}
     disabled={values.disabled}
     active={values.active}
-    size={(values.size = values.size || 'default')}
+    size={values.size}
```

Comment:
Ahhh.... got it 😁

Reply:
I passed in the mandatory option to the select so that values.size cannot be falsy